packages/wonder-blocks-dropdown/src/components/__tests__/multi-select.test.tsx

```diff
@@ -1668,8 +1671,17 @@ describe("MultiSelect", () => {
     });
 
     describe("a11y > Live region", () => {
-        beforeEach(() => {
-            initAnnouncer({debounceThreshold: 0});
+        let announceMessageSpy: any;
+
+        beforeAll(() => {
+            announceMessageSpy = jest.spyOn(
+                require("@khanacademy/wonder-blocks-announcer"),
+                "announceMessage",
+            );
```

Comment:
If I'm understanding correctly, we spy on the `announceMessage` function, which is also mocked out using `jest.mock(...)`. For my own learning, are both needed? Does one cover specific things the other doesn't?

Reply:
I mocked it and spied on it so I could replace the implementation of `announceMessage`. But it turned out I didn't need to go that far for the tests to pass. I removed the mocking part and it still works!